scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/task/internal/InternalTask.java

```diff
@@ -1196,6 +1197,8 @@ protected TaskLauncherInitializer getDefaultTaskLauncherInitializer() {
                                                              getId(),
                                                              internalJob.getSynchronizationAPI()));
 
+        tli.setSignalAPI(new SignalApiImpl(internalJob.getOwner(), getId(), internalJob.getSynchronizationAPI()));
```

Comment:
do you throw a signal execption on the method ?

Reply:
no, no need to do that